<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thirdDialog</class>
 <widget class="QDialog" name="thirdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#thirdDialog {
border-image :url(:/ultimate_submit.png)
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>1111</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
background-color: rgb(0, 85, 127);
}</string>
   </property>
   <widget class="QPushButton" name="Upload">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Upload{
background-color: rgb(255, 255, 255);
border-width:0px;
border-radius:4px;
}
#Upload::hover{
background-color:rgb(115, 255, 105);

}</string>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>380</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#submit2{
background-color: rgb(255, 255, 255);
font: 75 16pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:4px;
}
#submit2::hover{
background-color:rgb(115, 255, 105);
font: 75 14pt &quot;Sitka Display&quot;;

}</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;upload a photo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>391</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5{
background-color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;label_propic&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_4{
	background-color: rgb(255, 255, 255);

font: 50 12pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:1px;

}
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#comboBox{
	selection-background-color: rgb(255, 255, 255);
	
	alternate-background-color: rgb(255, 255, 255);
	background-color: rgb(255, 255, 255);
font: 50 12pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:1px;

}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_3{
	background-color: rgb(255, 255, 255);
font: 50 12pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:1px;

}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit{
	background-color: rgb(255, 255, 255);
font: 50 12pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:1px;

}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_2{
	background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:1px;

}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="choosepblm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>199</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Choose Your Problem:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>60</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;Title :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>94</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Location:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>124</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;Description:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;Contract Info:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>86</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;Login as :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="signout2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#signout2{
background-color: rgb(255, 255, 255);
font: 75 16pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:4px;
}
#signout2:hover{
background-color:rgb(115, 255, 105);
font: 75 14pt &quot;Sitka Display&quot;;

}</string>
   </property>
   <property name="text">
    <string>Sign out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newsfeed">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#newsfeed{
background-color: rgb(255, 255, 255);
font: 75 16pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:4px;
}
#newsfeed:hover{
background-color:rgb(115, 255, 105);
font: 75 14pt &quot;Sitka Display&quot;;

}</string>
   </property>
   <property name="text">
    <string>Newsfeed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
